index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+<style>
+
+  img{
+    display:block;
+    max_width:100%;
+    
+
+  }
+  #image{
+     max-width: 100%;
+    width:100%;
+  }
+</style>
+  <body >
+   <main id="main">
+     <h1 id = "title">Dr. Norman Borlaug</h1>
+        <p> The man who saved a billion lives</p>  
```

Comment:
I would suggest you use a <h2> element instead of the <p> tag. This is because it is a title and not a paragraph